### ISSUE TITLE
Add more output options for ecologits results

### ISSUE THREAD

**Issue by Neyri** [genai-impact/ecologits#118]
### Description

Hi,
Thank you for the greatwork performed on this library.
Similar to [CodeCarbon](https://mlco2.github.io/codecarbon/output.html#output), I would like to be able to send all collected data and environmental impacts to a central database. 
Would you consider adding support for API calls or automatic logging ?

**Comment by samuelrince** [comment]
Hello @Neyri, thanks for asking, this is definitely something that we have in mind! For now, we have not started any work on this particular topic, but will soon!

**Comment by eharris128** [comment]
@Neyri did you ever dive further into this?

**Comment by Neyri** [comment]
> @Neyri did you ever dive further into this?

Unfortunately I did not. We were able to integrate it into some of langfuse monitoring but we had to overload some of their monitoring process

**Comment by eharris128** [comment]
Yeah this sounds about right.

I am thinking that I need to write an abstraction of their methodology because I am working with post-chat completion token I/O counts.

Do not really have the chance to hook into the `chat.completions` chain.

An API would be wonderful here.

Was the work you did open source or no?

I know very little about carbon / environmental impact monitoring but I think it is important to enable developers of tools such as https://github.com/ryoppippi/ccusage who are taking advantage of claude code to better understand their environmental impact.

**Comment by eharris128** [comment]
How goes the work?

Also did you all see - https://mistral.ai/news/our-contribution-to-a-global-environmental-standard-for-ai - I am going to very naively use their numbers as a proxy for now :)
